scimath/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ Functionals/FunctionFactoryErrors.cc
 Functionals/FunctionalProxy.cc
 Functionals/SerialHelper.cc
 Mathematics/Combinatorics.cc
-Mathematics/FFTPack.cc
+#Mathematics/FFTPack.cc
```

Comment:
To be deleted (commenting out will likely only confuse since the file is gone).

Reply:
indeed; removed in new commit